sys/signal.c

```diff
@@ -0,0 +1,200 @@
+#include <signal.h>
+#include <thread.h>
+#include <malloc.h>
+#include <stdc.h>
+#include <errno.h>
+#include <sysent.h>
+#include <systm.h>
+#include <proc.h>
+
+static sighandler_t *signal_default_actions[SIG_LAST] = {
+    [SIGINT] = SIG_TERM,  [SIGILL] = SIG_TERM,  [SIGFPE] = SIG_TERM,
+    [SIGABRT] = SIG_TERM, [SIGSEGV] = SIG_TERM, [SIGKILL] = SIG_TERM,
+    [SIGCHLD] = SIG_IGN,  [SIGUSR1] = SIG_TERM, [SIGUSR2] = SIG_TERM,
+};
```

Comment:
We have at least four different default actions:
* `SA_KILL` terminate the process,
* `SA_IGNORE` ignore the signal,
* `SA_STOP` stop the process,
* `SA_CONT` continue the process if it is currently stopped.

Signal actions and properties are listed here [signalvar.h](http://bxr.su/NetBSD/sys/sys/signalvar.h#95).